### PR TITLE
refactor(organisms) SfGroupedProduts

### PR DESCRIPTION
**Pull request by aniamusial**
# Scope of work
- adjusted stories - replaced color with SfProperty
- fixed fonts and spacers
- reorganising content requires changes in template -> moved to another issue

# Checklist

- [ ] No commented blocks of code left
- [ ] Run tests and docs
- [ ] Self code-reviewed

### CODE REVIEW

File: packages/shared/styles/components/organisms/SfGroupedProduct.scss [hunk 1]
```diff
@@ -2,16 +2,13 @@
...
-      var(--font-size-regular),
-      1.6,
-      var(--font-family-secondary)
-    )
+    var(--font-normal),
+    var(--font-2xs),
+    1.6,
+    var(--font-family-secondary)
   );
   border-bottom: var(--grouped-product-item-border-bottom);
```
Comment (martaradziszewska) [inline packages/shared/styles/components/organisms/SfGroupedProduct.scss]:
```suggestion
@include border(--grouped-product-item-border, 0, solid, var(--c-light));
```

File: packages/shared/styles/components/organisms/SfGroupedProduct.scss [hunk 2]
```diff
@@ -26,34 +23,29 @@
...
     margin: var(--grouped-product-item-price-margin);
     font: var(--grouped-product-item-price-font);
   }
   @include for-desktop {
-    --input-background: var(--c-light);
-    --input-font-size: var(--font-size-big);
-    --grouped-product-item-font-size: var(--font-size-extra-small);
     --grouped-product-item-display: flex;
     --grouped-product-item-padding: 10px 0;
     --grouped-product-item-border-bottom: 1px solid var(--c-light);
```
Comment (martaradziszewska) [inline packages/shared/styles/components/organisms/SfGroupedProduct.scss]:
```suggestion
    --grouped-product-item-border-width: 0 0 1px 0;
```

File: packages/shared/styles/components/organisms/SfGroupedProduct.scss [hunk 3]
```diff
@@ -2,16 +2,13 @@
...
-      1.6,
-      var(--font-family-secondary)
-    )
+    var(--font-normal),
+    var(--font-2xs),
+    1.6,
+    var(--font-family-secondary)
   );
   border-bottom: var(--grouped-product-item-border-bottom);
   &__aside {
```
Comment (martaradziszewska) [inline packages/shared/styles/components/organisms/SfGroupedProduct.scss]: please add to aside `line-height: 0` to even bottom line

File: packages/shared/styles/components/organisms/SfGroupedProduct.scss [hunk 4]
```diff
@@ -26,34 +23,29 @@
...
   &__description {
     margin: var(--grouped-product-item-description-margin);
   }
   &__title {
-    margin: var(--grouped-product-item-title-margin, var(--spacer-small) 0);
+    margin: var(--grouped-product-item-title-margin, var(--spacer-xs) 0);
   }
   &__price {
     margin: var(--grouped-product-item-price-margin);
     font: var(--grouped-product-item-price-font);
```
Comment (martaradziszewska) [inline packages/shared/styles/components/organisms/SfGroupedProduct.scss]: are we need this?

**Comment by pspaczek** [comment]
Zrzut ekranu 2020-03-23 o 17 03 21
Zrzut ekranu 2020-03-23 o 17 03 10

**Pull request merged**